.coveragerc

```diff
@@ -11,9 +11,21 @@ omit =
     distributed/utils_test.py
     distributed/deploy/ssh.py
     distributed/_ipython_utils.py
+    distributed/dashboard/components/nvml.py
+    distributed/_version.py
+    distributed/pytest_resourceleaks.py
+    distributed/comm/ucx.py
 
 [report]
 show_missing = True
+exclude_lines =
+    # re-enable the standard pragma
+    pragma: no cover
+    # exclude nvml calls
+    \snvml\.
```

Comment:
I think this should actually be something like `[\s(.]nvml\.` (and same for pynvml)